Sources/Parser/Parser+Declaration.swift

```diff
@@ -251,24 +268,49 @@ extension Parser {
   func parseContractBehaviorMembers(contractIdentifier: RawTypeIdentifier) throws -> [ContractBehaviorMember] {
     var members = [ContractBehaviorMember]()
 
-    while true {
-      let attrs = try parseAttributes()
-      let modifiers = try parseModifiers()
+    while let first = currentToken?.kind {
+      switch first {
+        case .func, .init, .fallback, .public, .visible, .mutating, .punctuation(.at):
```

Comment:
What is `@mutating` for? Also I think this is missing identifiers for the contract behaviour declaration block.

Reply:
E.g. a function:
```
mutating func ....
```
and 
```
@payable
func ...
```

Reply:
This is a contract behaviour member so you can't have nestled contract behaviours in contract behaviours.

Reply:
Yes, it's a contract behaviour member..., and I thought mutating was an annotation for some reason, I remember it's a modifier now. :|